amplfi/train/models/flow.py

```diff
@@ -222,6 +273,8 @@ def cast_as_bilby_result(
             samples:
                 An array of posterior samples of shape
                 (1, num_samples, num_params)
+            log_probs:
+
```

Comment:
missing docstring for `log_priors` (please rename this to log_probs_prior)

Reply:
done